.github/workflows/codeql.yml

```diff
@@ -47,21 +47,22 @@ jobs:
           sudo apt-get install meson
 
     # Initializes the CodeQL tools for scanning.
-    - name: Initialize CodeQL
+    - if: matrix.language == 'c-cpp'
+      name: Initialize CodeQL C
       uses: github/codeql-action/init@v2
       with:
-        languages: ${{ matrix.language }}
-        # If you wish to specify custom queries, you can do so here or in a config file.
-        # By default, queries listed here will override any specified in a config file.
-        # Prefix the list here with "+" to use these queries and those in the config file.
-
-        # For more details on CodeQL's query packs, refer to: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#using-queries-in-ql-packs
-        # queries: security-extended,security-and-quality
-
+        languages: 'c-cpp'
 
+    - if: matrix.language == 'python'
+      name: Initialize CodeQL Python
+      uses: github/codeql-action/init@v2
+      with:
+        languages: 'python'
+        config-file: ./.github/codeql/codeql-config.yml
+  
     - name: meson build
       run: |
-         meson setup --force-fallback-for=libnvme .build
+         meson setup --force-fallback-for=libnvme,json-c .build
```

Comment:
I think we should also enable the OpenSSL library here (not sure if's the right place though) because I think it's properly the most interesting thing to cover for security holes.

Reply:
Or instead the fallbacks we could try to use the pre-build container. Just as an idea.

Reply:
I'll try adding openssl at this point and see what happens after a run through codeql. 

I don't know enough (currently) about the pre-build container approach to comment convincingly, but it sounds interesting :-).

Reply:
Sorry for the confusion. I was not interested to check OpenSSL just the code inside nvme-cli which uses OpenSSL. But thinking again on it, the interesting bits are in libnvme anyway. So this is not really needed after all...  Sorray abou that too.

Reply:
No problem at all - understood. I've backed out the change to include openssl as a fallback.